mlkem/verify.h

```diff
@@ -237,9 +240,9 @@ __contract__(
 {
   uint8_t r = 0, s = 0;
 
-  // Switch to a _signed_ ilen value, so that our loop counter
-  // can also be signed, and thus (i - 1) in the loop invariant
-  // can yield -1 as required.
+  /* Switch to a _signed_ ilen value, so that our loop counter */
+  /* can also be signed, and thus (i - 1) in the loop invariant */
```

Comment:
Ideally, those should be a single block comment.

Reply:
resolved now.